model_transactions.go

```diff
@@ -1001,3 +1023,65 @@ func processTransaction(ctx context.Context, transaction *Transaction) error {
 
 	return transaction.Save(ctx)
 }
+
+// getTransactionsByConditions will get the sync transactions to migrate
+func getTransactionsByConditions(ctx context.Context, conditions map[string]interface{},
+	queryParams *datastore.QueryParams, opts ...ModelOps,
+) ([]*Transaction, error) {
+	if queryParams == nil {
+		queryParams = &datastore.QueryParams{
+			OrderByField:  createdAtField,
+			SortDirection: datastore.SortAsc,
+		}
+	} else if queryParams.OrderByField == "" || queryParams.SortDirection == "" {
+		queryParams.OrderByField = createdAtField
+		queryParams.SortDirection = datastore.SortAsc
+	}
+
+	test := NewBaseModel(ModelNameEmpty, opts...)
+	test2 := test.Client()
+	test3 := test2.Datastore()
+	fmt.Println(test3)
```

Comment:
Should this be here?

Reply:
I am sorry, my bad. Of course not. Fixed